sdk/eventhub/Azure.Messaging.EventHubs.Processor/tests/EventProcessorClient/EventProcessorClientLiveTests.cs

```diff
@@ -8,13 +8,10 @@
 using System.Text;
 using System.Threading;
 using System.Threading.Tasks;
-using Azure.Messaging.EventHubs.Core;
-using Azure.Messaging.EventHubs.Processor;
-using Azure.Messaging.EventHubs.Processor.Tests;
-using Azure.Messaging.EventHubs.Tests.Infrastructure;
+using Azure.Messaging.EventHubs.Tests;
 using NUnit.Framework;
 
-namespace Azure.Messaging.EventHubs.Tests
+namespace Azure.Messaging.EventHubs.Processor.Tests
```

Comment:
I don't have a strong opinion, but if the classes are in the `Azure.Messaging.EventHubs` namespace, wouldn't we expect the tests to be in `Azure.Messaging.EventHubs.Tests`?

Reply:
You have a point, but should the classes be in the `Azure.Messaging.EventHubs` namespace? I would have put them in the `Processor` namespace if I noticed that.